src/components/MovieLibrary.jsx

```diff
@@ -1 +1,36 @@
 // implement MovieLibrary component here
+import React, { Component } from 'react';
+
+import MovieList from './MovieList';
+import SearchBar from './SearchBar';
+import AddMovie from './AddMovie';
+
+class MovieLibrary extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchTest: '',
```

Comment:
Errinho de digitação aqui Leléo:
```suggestion
      searchText: ' ',
```